launcher/modplatform/packwiz/Packwiz.h

```diff
@@ -88,6 +88,9 @@ class V1 {
      * If the mod doesn't have a metadata, it simply returns an empty Mod object.
      * */
     static auto getIndexForMod(const QDir& index_dir, QVariant& mod_id) -> Mod;
+
+    /* Returns if version a is less than version b, ie: 1.20 < 1.20.1; 1.20.2 < 1.20.10*/
+    static bool versionsSort(QString a, QString b);
```

Comment:
Also, I'd remove it from this class (you could also make it private but you can just have the function only in the cpp)

Reply:
Because now it is just a one-line function, I would suggest just using it as a lambda instead